--- /opt/glite/yaim/functions/config_vobox.orig 2020-06-21 21:24:30.088433160 +0000
+++ /opt/glite/yaim/functions/config_vobox      2020-06-21 21:24:53.868433849 +0000
@@ -61,13 +61,13 @@
 
     vodir=${VOBOX_PROXY_ROOT}/${i}
     mkdir -p $vodir
-    chown ${ACCOUNT}. $vodir
+    chown ${ACCOUNT}: $vodir
 
     for dir in start stop agents info-provider log proxy_repository; do
       d=$vodir/${dir}
       mkdir -p $d
       chmod 700 $d
-      chown ${ACCOUNT}. $d
+      chown ${ACCOUNT}: $d
     done
 
     # install the renewal service